ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -431,7 +455,17 @@ export default React.memo<Props>(function VideoJs(props: Props) {
 
     // Cleanup
     return () => {
-      window.removeEventListener('keydown', curried_function);
+      const volumePanel = playerRef.current
+        .getChild(VIDEOJS_CONTROL_BAR_CLASS)
+        .getChild(VIDEOJS_VOLUME_PANEL_CLASS)
+        .el();
+
+      window.removeEventListener('keydown', keyDownHandlerRef.current);
+      containerRef.current.removeEventListener('wheel', videoScrollHandlerRef.current);
```

Comment:
![image](https://user-images.githubusercontent.com/64950861/171785285-127b5e82-5a74-4c69-bac2-ad1b7186113c.png)

Just FYI for now -- can leave as-is, because ...

We plan to move the keyboard shortcut code into a plugin so that we don't need to inject these kind of changes here, which has nothing to do with the effect and violates the 1-thing-per-effect general rule. 

Inside the plugin, we can use the vjs framework to listen to these events instead of attaching the listeners ourselves.
```
player.on('mousewheel', mouseScroll);
componentXYZ.on('mousewheel', mouseScroll);
```

But can stay with this approach until then...